PhysicsTools/NanoAOD/python/extraflags_cff.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+
+# Bad/clone muon filters - tagging mode to keep the event
+from RecoMET.METFilters.badGlobalMuonTaggersMiniAOD_cff import badGlobalMuonTaggerMAOD, cloneGlobalMuonTaggerMAOD
+badGlobalMuonTagger = badGlobalMuonTaggerMAOD.clone(
+    taggingMode = True
+)
+
+cloneGlobalMuonTagger = cloneGlobalMuonTaggerMAOD.clone(
+    taggingMode = True
+)
+
+# Bad charge hadron
+from RecoMET.METFilters.BadChargedCandidateSummer16Filter_cfi import BadChargedCandidateSummer16Filter
+BadChargedCandidateTagger = BadChargedCandidateSummer16Filter.clone(
```

Comment:
not clear to me if the right version to use is BadChargedCandidateSummer16Filter or BadChargedCandidateFilter, but from the comments in https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py#L10-L11 I would think it's the latter.

Reply:
it is the latter, the full flag list is also kept track of here: https://twiki.cern.ch/twiki/bin/view/CMS/MissingETOptionalFiltersRun2